Source/SwiftLintFramework/Models/LinterCache.swift

```diff
@@ -15,30 +15,35 @@ internal enum LinterCacheError: Error {
 }
 
 public final class LinterCache {
-    private var cache = [String: Any]()
+    private var readCache = [String: Any]()
+    private var writeCache = [String: Any]()
```

Comment:
Changing `readCache` and `writeCache` from `[String: Any]` to `[String: [String: Any]]` boosted performance on my test.